lux/test/integration/discord_message_lens_test.exs

```diff
@@ -0,0 +1,270 @@
+defmodule Lux.Integration.DiscordMessageLensTest do
+  use IntegrationCase, async: true
+
+  alias Lux.Lenses.Discord.MessageLens
+
+  @test_channel_id "123456789012345678"
+
+  setup do
+    Req.Test.verify_on_exit!()
```

Comment:
If this is an integration test, we should not be using Req. Integration tests actually hit the main service we want to test the integration for.